### PR TITLE
README updated with manual

### PR DESCRIPTION
**Pull request by oskarjerzyk**

### CODE REVIEW

**Comment by Mateusz512** [comment]
`mvn` should be in prerequisites as well

File: README.md [hunk 1]
```diff
@@ -2,20 +2,40 @@
...
 The repository allows to re-build all Knot.x modules and use them during integration tests, bypassing the
 need to publish artifacts to the maven repository first.
 
+After successful build remember to rebuild gradle modules or simply refresh them in your IDE
+
+## Verify if everything is up and running
+From [here](https://github.com/Knotx/knotx-example-project) you can download example Knot.x projects.
+For our training purposes you can use getting-started project which is contained in this repository.
+Navigate to them and build it. Do all the followings from ubuntu terminal. Executing it from Windows Command Prompt
+will fail with high probability. To check if this example projects was built properly, just open link
```
Comment (Mateusz512) [inline README.md]: `will fail with high probability` sounds bad, we should just provide an overall info that we do not support deployment on Windows machines

File: README.md [hunk 2]
```diff
@@ -2,20 +2,40 @@
 This section describes tools that help you with coding. We provide scripts that allows to
 clone all repositories and install artifacts to Maven local / Maven Central Snapshot repositories.
 
+## Prerequisites on Windows
+- Install and configure WSL - Windows Subsystem for Linux - it's really necessary, without it
```
Comment (tomaszmichalak) [inline README.md]: Install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL) and configure:
- JDK 8
- Gradle 5+
- Maven

File: README.md [hunk 3]
```diff
@@ -2,20 +2,49 @@
...
+After successful build remember to rebuild gradle modules or simply refresh them in your IDE
+
+## Run Stack
+From [here](https://github.com/Knotx/knotx-example-project) you can download example Knot.x projects.
+Let's use `getting-started` project which is contained in this repository as an example.
+Navigate to the project directory and build it 
+```
+gradlew clean build
+``` 
+After successful build you'll find zip archive `knotx-stack-2.0.0-SNAPSHOT.zip` under `./build/distributions`. Unzip it
```
Comment (Voycawojka) [inline README.md]: It should be noted that the knotx version can be different in the future

File: README.md [hunk 4]
```diff
@@ -2,20 +2,49 @@
...
+Install and configure [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL) and configure:
+- Java 8 on WSL
+- Gradle 5+
+- Maven
+
 ## Clone all repositories
 Check a `development/pull-all.sh` script to clone all Knot.x repositories. Please check `-h` option
 for help. Execute this script from `knotx-aggregator/development` directory, executing it from another 
-directory will fail.
+directory will fail. Please check `-r` option for providing custom directory where Knot.x repositories
```
Comment (tomaszmichalak) [inline README.md]: The `-r` parameter specifies a directory all Knot.x repositories are cloned, such as `sh pull-all.sh -r ../..`.

File: README.md [hunk 5]
```diff
@@ -2,20 +2,49 @@
...
 Check a `development/pull-all.sh` script to clone all Knot.x repositories. Please check `-h` option
 for help. Execute this script from `knotx-aggregator/development` directory, executing it from another 
-directory will fail.
+directory will fail. Please check `-r` option for providing custom directory where Knot.x repositories 
+will be cloned, for example `sh pull-all.sh -r ../../knotx` make sure, that provided directory exists. 
 
 ## Build Stack
 Check a `development/build-stack.sh` script to build all cloned repositories. Please check `-h` option
-for help.
+for help. This command should also be executed from `knotx-aggregator/development` directory, in other
```
Comment (tomaszmichalak) [inline README.md]: Execute this command from the `knots-aggregator/development` directory. Specify the `-r` parameter to point the directory with cloned repositories:
```
sh build-stack.sh -r ../..
```

File: README.md (hunk 4 above)
Comment (tomaszmichalak) [reply README.md]: In this case the pointed directory exists for sure ;)

File: README.md [hunk 6]
```diff
@@ -2,20 +2,49 @@
...
+case it will fail. Please check `-r` option for building Knot.x repositories in custom location, for example
+`sh buils-stack.sh -r ../../knotx` make sure that in the provided directory repositories are properly
+cloned.
 
 Please note that the `knotx-stack` repository contains
 [Gradle composite build](https://docs.gradle.org/current/userguide/composite_builds.html) definition.
 The repository allows to re-build all Knot.x modules and use them during integration tests, bypassing the
 need to publish artifacts to the maven repository first.
 
+After successful build remember to rebuild gradle modules or simply refresh them in your IDE
```
Comment (tomaszmichalak) [inline README.md]: The `build-all.sh` command deploys the [Knot.x BOM file](https://github.com/Knotx/knotx-dependencies) (Bill Of Materials)  to the local Maven repository first. The BOM file specifies external dependencies, such as used Vert.x version etc. Then all modules are rebuilt.
After the execution, please refresh your IDE.

Comment (oskarjerzyk) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (oskarjerzyk) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (oskarjerzyk) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (oskarjerzyk) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (oskarjerzyk) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (oskarjerzyk) [reply README.md]: fixed

**Review by malaskowski — APPROVED** [review]

**Pull request merged**